packages/kit/src/runtime/server/page/render.js

```diff
@@ -297,11 +297,19 @@ export async function render_response({
 
 		const blocks = [];
 
-		const properties = [
-			paths.assets && `assets: ${s(paths.assets)}`,
-			`base: ${base_expression}`,
-			`env: ${!client.uses_env_dynamic_public || state.prerendering ? null : s(public_env)}`
-		].filter(Boolean);
+		// when serving a prerendered page in an app that uses $env/dynamic/public, we must
```

Comment:
Can we add the Safari bug link in here somewhere + a "TODO" tag? It'd be a nice callout for us when we're doing our next major release.

Reply:
done